### ISSUE TITLE
Display problem using iPad

### ISSUE THREAD

**Issue by mupendara** [litomba/www#153]
## Steps to reproduce / User Story
1. Open Safari using an iPad
2. Display litomba landing page

## Environment
Production

### Operating System
- [ ] iOS

### Browser
- [ ] Safari

## Notes
Check the screenshot
![ipad_error1](https://user-images.githubusercontent.com/30872324/30039665-49f15db6-91cc-11e7-9623-94a830a68ef0.jpeg)

**Comment by mupendara** [comment]
![ipad_error3](https://user-images.githubusercontent.com/30872324/30039678-7cd13788-91cc-11e7-84bf-3bfaefe612bf.jpeg)

**Comment by AntonioVdlC** [comment]
This is the exact same problem you had with Safari on Windows ... it's probably a very old version that is not even supported by Apple anymore.

I have mixed feelings about this, because on one hand Apple officially doesn't support that version of Safari (so why should we? 😅 ), but on the other hand if we have an important number of users using that version then we'll have to support it 🤔 .

Can we have a percentage of users using Safari and its different versions?

**Comment by zuluzaire** [comment]
I agree with you, let's t least display an error message informing the user that litomba is not adapted to the browser version he uses because it's too old. And we display the minimum version adapted to litomba.

What you think ?

**Comment by zuluzaire** [comment]
Regarding the percentage of users using safari and their version, we will need to monitor it on mixpanel.
For now you will find the figures below:
![image](https://user-images.githubusercontent.com/30872324/30153391-57a0b4d2-93ae-11e7-9ffe-5dea4e3d0fcc.png)


![image](https://user-images.githubusercontent.com/30872324/30153326-1c7b220c-93ae-11e7-98b7-d9d3968d04d9.png)

**Comment by AntonioVdlC** [comment]
Ok, so apparently, Safari 8 needs a `-webkit-` prefix to be able to understand flexbox ... let's keep this in the backlog for now!
Solving this means I'll need to eject the creat-react-app scripts, which is feasible, but a bit of a pain right now! 😅
